viime/views.py

```diff
@@ -279,6 +280,27 @@ def set_csv_file_description(csv_id, description):
         raise
 
 
+@csv_bp.route('/csv/<uuid:csv_id>/group-levels', methods=['PUT'])
+@use_kwargs({
+    'group_levels': fields.List(webargs.fields.Nested({
+        'name': fields.Str(required=True),
+        'label': fields.Str(required=True),
+        'description': fields.Str(missing=None),
+        'color': fields.Str(required=True),
+    }), required=True)
```

Comment:
webargs works with marshmallow, so I think you can replace this with:
`fields.List(GroupLevelSchema(), required=True)`

Reply:
the `GroupLevelSchema` requires a `csv_file_id` which is not given by the request parameters

Reply:
How about `GroupLevelSchema(exclude=['csv_file_id'])`?